aiida/transports/plugins/ssh.py

```diff
@@ -68,92 +68,97 @@ class SshTransport(Transport):  # pylint: disable=too-many-public-methods
     # I disable 'password' and 'pkey' to avoid these data to get logged in the
     # aiida log file.
     _valid_connect_options = [
-        ('username', {
-            'prompt': 'User name',
-            'help': 'user name for the computer',
-            'non_interactive_default': True
-        }),
+        (
+            'username', {
+                'prompt': 'User name',
+                'help': 'Login user name on the remote machine.',
+                'non_interactive_default': True
+            }
+        ),
         ('port', {
             'option': options.PORT,
-            'prompt': 'port Nr',
+            'prompt': 'Port number',
             'non_interactive_default': True
         }),
         (
             'look_for_keys', {
                 'switch': True,
                 'prompt': 'Look for keys',
-                'help': 'switch automatic key file discovery on / off',
+                'help': 'Automatically look for private keys in ~/.ssh (default: true)',
                 'non_interactive_default': True
             }
         ),
         (
             'key_filename', {
                 'type': AbsolutePathOrEmptyParamType(dir_okay=False, exists=True),
                 'prompt': 'SSH key file',
-                'help': 'Manually pass a key file if default path is not set in ssh config',
+                'help': 'Absolute path to your private SSH key. Leave empty to use path set in ssh config.',
                 'non_interactive_default': True
             }
         ),
         (
             'timeout', {
                 'type': int,
                 'prompt': 'Connection timeout in s',
-                'help': 'time in seconds to wait for connection before giving up',
+                'help': 'Time in seconds to wait for connection before giving up. Leave empty to use default value.',
                 'non_interactive_default': True
             }
         ),
         (
             'allow_agent', {
                 'switch': True,
                 'prompt': 'Allow ssh agent',
-                'help': 'switch to allow or disallow ssh agent',
+                'help': 'Switch to allow (default) or disallow using an SSH agent.',
```

Comment:
as in remove `'swith: True` and alter the help text?

Reply:
As in remove 'switch': True` and alter the help text?

Reply:
I see now, didn't realize it was a switch. Honestly, doesn't the flag `--disallow-agent`  make more sense here, instead of a switch? Anyway, probably shouldn't be changing this now because it would be backwards incompatible with scripts. But maybe good to open an issue

Reply:
I am trying to think what the disadvantages of a flag would be and the only thing I can come up with is, if at some point we want to change the default behavior, we would have to change the flag name, e.g., if in the future we decide it is better to always disallow the agent by default, we would have to rename the flag to `--allow-agent`.

Reply:
In click you can just add `--allow-agent/--disallow-agent` and set the default. Not sure how that integrates with the code here?

Reply:
see #4226